runtime/compiler/optimizer/J9EstimateCodeSize.hpp

```diff
@@ -77,6 +78,7 @@ class TR_J9EstimateCodeSize : public TR_EstimateCodeSize
 
    protected:
       bool estimateCodeSize(TR_CallTarget *, TR_CallStack * , bool recurseDown = true);
+      TR::CFG &generateCFG(TR_CallTarget *calltarget, TR::Region &cfgRegion, TR_J9ByteCodeIterator &bci, NeedsPeekingHeuristic *nph, TR::Block** blocks, flags8_t * flags, int &size, bool& thisOnStack, bool& hasThisCalls, bool& unresolvedSymbolsAreCold);
```

Comment:
Could you add some doxygen comments to briefly explain what this function does and what are some of the important arguments?